.github/workflows/close-stale-issues.yml

```diff
@@ -0,0 +1,34 @@
+name: "Close Stale Issues"
+
+# Controls when the action will run.
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 */4 * * *"
+
+jobs:
+  cleanup:
+    permissions:
+      issues: write
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    name: Stale issue job
+    steps:
+      - uses: actions/stale@v8
+        with:
+          issue-types: issues
```

Comment:
I don't see this field on https://github.com/actions/stale?tab=readme-ov-file

We could artificially skip PRs by using the `only-pr-labels` field with a non-existing label.